README.md

```diff
@@ -129,6 +129,43 @@ To use a `<span>` or `<p>` element, use the following:
 <Text.p>This is a p element</Text.p>
 ```
 
+### `<Heading />`
+
+Use the `<Heading />` component to use HTML heading h1-h6 element, with all the `<Text>` properties.
```

Comment:
For React docs we should call them `props` and not `properties` since that's how React refers to them

Reply:
Updated the terminology of `properties` to `props` related to `Heading` topic in the `README.md` and the `Heading` story.